### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by olschulz**
add localization for protected routes and fix logo button not working on startpage

### CODE REVIEW

**Comment by rheaSPK** [comment]
Bitte eine kurze Beschreibung geben, was gemacht wurde, damit PR nicht leer sind

**Comment by olschulz** [comment]
> Bitte eine kurze Beschreibung geben, was gemacht wurde, damit PR nicht leer sind

🙃

**Review by rheaSPK — APPROVED** [review]
Otherwise makes sense and looks good

File: projects/bp-gallery/src/components/common/ScrollContainer.tsx [hunk 1]
```diff
@@ -1,17 +1,33 @@
...
+  const history: History = useHistory();
+
+  //neccessary so that container scrolls to top when pressing on top bar logo on startpage
+  useEffect(() => {
+    if (scrollTop && history.location.state?.scrollTop) {
+      scrollDiv.current?.scrollTo(0, 0);
+      history.replace('', { showBack: false, scrollTop: false });
+    }
+    // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
+  }, [scrollTop, history, history?.location?.state]);
```
Comment (MariusDoe) [inline projects/bp-gallery/src/components/common/ScrollContainer.tsx:25]: I'm not sure we need this. If I understand correctly, this only applies to startpage-to-startpage navigation, every other navigation works fine regarding scrolling, right? If so, I think I would be fine to just leave the scroll position as is when "navigating" from the startpage to the startpage, i. e. we could remove this code.

**Review by MariusDoe — CHANGES_REQUESTED** [review]

File: projects/bp-gallery/src/components/common/ScrollContainer.tsx (hunk 1 above)
Comment (olschulz) [reply projects/bp-gallery/src/components/common/ScrollContainer.tsx]: done

**Review by MariusDoe — APPROVED** [review]

**Pull request merged**